### ISSUE TITLE
Defining a Scheme for each DOF

### ISSUE THREAD

**Issue by josep-m-carbonell** [KratosMultiphysics/GiDInterface#431]
In the Solid and Pfem application the definition of the strategy must consider that the configuration of the scheme is posterior to the selection of the Elements, once the DOF have been defined. In a monolithic strategy the convegence criterion and the scheme will coincide, but in a segregated strategy we need to define a scheme, a convergence criterion and a linear solver for each segregated set of DOFs.

How can be define this structure in the Strategies.xml ?

**Comment by jginternational** [comment]
That is not possible in this moment. 
You can set a different linear solver for different variables, like Fluid Dynamics app does:
- One for Velocity
- One for Pressure

This is achieved in the Strategies.xml file, in the Linear solver configuration section. You can copy it from the Fluid.

**Comment by josep-m-carbonell** [comment]
Yes I knew that about the fluids, but it is a particular option no general enough to face more complex formulations. I must ask for a more general strategy again. May be some people of the pfem team @KratosMultiphysics/pfem  can help in programming it, me included.

**Comment by jgonzalezusua** [comment]
Me and @ipouplana can help in that and other stuff.
